metrics_utility/test/gather/test_gather_ranges.py

```diff
@@ -0,0 +1,55 @@
+import glob
+import os
+
+import pytest
+
+from metrics_utility.test.util import run_gather_ext
+
+
+env_vars = {
+    'METRICS_UTILITY_REPORT_TYPE': 'CCSPv2',
+    'METRICS_UTILITY_SHIP_PATH': './metrics_utility/test/test_data',
+    'METRICS_UTILITY_SHIP_TARGET': 'directory',
+    'METRICS_UTILITY_MAX_GATHER_PERIOD_DAYS': '3',
```

Comment:
still missing the test for 0?

Where 0 will really work only for collections doing snapshot of state (that means the collection itself doesn't use since/until in the query)